modules/ROOT/pages/mock-event-processor.adoc

```diff
@@ -45,9 +45,9 @@ This Dataweave file creates the payload to send in a POST request. You can then
 </munit-tools:then-return>
 ----
 
-Finally, you can configure a `then-return` element to define the type of response the mocked processor should return. It could be a payload, a variable, attributes, or even an error.
+Finally, you can configure a *then-return* element to define the type of response the mocked processor should return. It could be a payload, a variable, attributes, or even an error.
```

Comment:
```suggestion
Finally, you can configure a *then-return* element to define the type of response the mocked processor should return. It could be a payload, a variable, a list of attributes, or even an error.
```